packages/protocol/scripts/foundry/make-release.ts

```diff
@@ -0,0 +1,803 @@
+/* eslint-disable no-console */
+import { LibraryAddresses } from '@celo/protocol/lib/bytecode'
+import { ASTDetailedVersionedReport } from '@celo/protocol/lib/compatibility/report'
+import { getCeloContractDependencies } from '@celo/protocol/lib/contract-dependencies'
+import { CeloContractName, celoRegistryAddress } from '@celo/protocol/lib/registry-utils'
+import { ForgeArtifact } from '@celo/protocol/scripts/foundry/ForgeArtifact'
+import { NULL_ADDRESS, eqAddress } from '@celo/utils/lib/address'
+import { existsSync, readJsonSync, readdirSync, writeJsonSync } from 'fs-extra'
+import { basename, join } from 'path'
+import { TextEncoder } from 'util'
+import {
+  Abi,
+  AbiParameter,
+  Account,
+  Chain,
+  Hex,
+  PublicClient,
+  Address as ViemAddress,
+  WalletClient,
+  createPublicClient,
+  createWalletClient,
+  encodeFunctionData,
+  http,
+  keccak256,
+  toHex,
+} from 'viem'
+import { mnemonicToAccount, privateKeyToAccount } from 'viem/accounts'
+import * as viemChains from 'viem/chains'
+import yargs from 'yargs'
+import { hideBin } from 'yargs/helpers'
+import { getReleaseVersion, ignoredContractsV9 } from '../../lib/compatibility/ignored-contracts-v9'
+
+interface MakeReleaseArgv {
+  report: string
+  proposal: string
+  librariesFile: string
+  initialize_data: string
+  build_directory: string
```

Comment:
Nice to have detail 💅 , but I would suggest to stick to camelCase
```suggestion
  initializeData: string
  buildDirectory: string
```

Reply:
when new npm package is introduced